source/opt/mem_pass.cpp

```diff
@@ -170,6 +179,11 @@ bool MemPass::IsLiveStore(ir::Instruction* storeInst) {
   // get store's variable
   uint32_t varId;
   (void)GetPtr(storeInst, &varId);
+  if (varId == 0) {
+    // If we do know now which variable we are accessing, assume the store is
```

Comment:
If we do *not* know which variable .... ?

Reply:
done.